Nearby Sharing Windows/Resources/values-ca/strings.xml

```diff
@@ -0,0 +1,96 @@
+<string name="bt_mac_address">Direcció MAC del Bluetooth</string>
+<!-- #endregion -->
+
+<!-- #region activity_receive.xml -->
+<string name="visible_as_template">Visible com {0}</string>	
+<string name="debug_log">Registre de depuració</string>
+<!-- #endregion -->
+
+<!-- #region activity_share.xml -->
+<string name="scanning">Escaneant...</string>
+<!-- #endregion -->
+
+<!-- #region preferences.xml -->
+<string name="category_general">General</string>
+<string name="category_links">Enllaços / Ajuda</string>
+<string name="category_permissions">Permisos</string>
+<string name="category_setup">Configuració inicial</string>
+<string name="category_about">Sobre</string>
+
+<string name="request_permissions_title">Sol·licitar  permisos necesaris</string>
+<string name="request_permissions_summary">Dona-li els permisos necessaris a l \' aplicació </string>
+
+<string name="open_sponsor_title">Patrocinar</string>
+<string name="open_sponsor_summary">Patrocina al creador de l \' aplicació</string>
+
+<string name="open_faq_title">Ajuda</string>
+<string name="open_faq_summary">Obrir les Preguntes Freqüents</string>
+        
+<string name="open_discord_title">Discord</string>
+<string name="open_discord_summary">Unir-se al servidor de Discord</string>
+
+<string name="preference_appearance_title">Interfície / Diseny</string>
+<string name="preference_appearance_summary">Idioma, tema fosc</string>
+
+<string name="force_dark_mode_title">Tema fosc</string>
+<string name="force_dark_mode_summary">Forçar el tema fosc</string>
+
+<string name="switch_language_title">Idioma de l\' aplicació</string>
+<string name="switch_language_summary">Cambiar l \' idioma de l \'aplicació</string>
+
+<string name="preference_cdp_title">Plataforma de dispositius conectats</string>
+<string name="preference_cdp_summary">Configurar l\' enviament i la recepció </string>
+
+<string name="open_setup_title">Configuración del PC</string>
+<string name="open_setup_summary">Tutorial sobre los ajustes del PC necesarios</string>
+
+<string name="goto_mac_address_title">Indica la direcció MAC</string>
+<string name="goto_mac_address_summary">Indica la direcció MAC d \' aquest dispositiu </string>
+
+<string name="device_name_title">Nom del dispositiu</string>
+<string name="device_name_summary">Canvia com et veu la resta</string>
+
+<string name="show_credits_title">Mostrar crèdits</string>
+<!-- #endregion -->
+	
+<!-- #region ReceiveActivity.cs -->
+<string name="receive_missing_permissions">No es pot rebre sense els permisos necessaris!</string>
+<!-- #endregion -->
+	
+<!-- #region ReceiveActivity.cs -->
+<string name="share_file">Compartir arxiu</string>
+<string name="share_url">Compartir URL</string>
+
+<string name="send_missing_permissions">Falten alguns permisos necessaris!</string>
+<string name="wait_for_acceptance">Esperant reposta...</string>
+<string name="sending_template">Enviant ... {0}/{1} arxius... {2}%</string>
```

Comment:
I changed this template
https://github.com/nearby-sharing/android/blob/6f84dfca6ad0fd0ce84d6d28d52d5a5bcc8cb9cc/src/Resources/values/strings.xml#L97